adapter/pkg/messaging/notificationListener.go

```diff
@@ -212,6 +232,7 @@ func handleSubscriptionEvents(data []byte, eventType string) {
 
 // handleScopeRelatedEvents to process scope related events
 func handleScopeEvents(data []byte, eventType string) {
+	// TODO: (VirajSalaka) Decide the usage
```

Comment:
Remove as it is unused.

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/1651/commits/34135e565e0feb892396850af240c35fdc8c4499